src/app/endround/endround.component.html

```diff
@@ -0,0 +1,31 @@
+<link rel="preload" as="image" href="../../assets/misc/endround-bg.webp" />
+<link rel="preload" as="image" href="../../assets/misc/logo.webp" />
+<div *ngIf="match.roundPhase === 'end'">
+  <div class="intro-box">
+    <div
+      class="intro-box-content"
+      [style.background-image]="'url(../../assets/misc/endround-bg.webp)'"
+    >
+      <div class="info-box">
+        <img class="info-image" src="../../assets/misc/logo.webp" />
+        <div class="round-number">ROUND WIN</div>
+      </div>
+    </div>
+  </div>
+  <div class="endround">
+    <div class="endround-box" [style.background-image]="'url(../../assets/misc/endround-bg.webp)'">
+      <div class="win-side">
+        <div class="win-side-image-box">
+          <img class="win-side-image" src="{{ match.teams[teamWon].teamUrl }}" />
+        </div>
+        <div class="win-side-text">{{ match.attackersWon ? "ATTACKER" : "DEFENDER" }} WIN</div>
+      </div>
+      <div class="info-box">
+        <div class="info-image-box">
+          <img class="info-image" src="../../assets/misc/logo.webp" />
```

Comment:
If you want to, you can add some logic to use "match->tournamentUrl" for an upcoming change where we add the tournament logo url to the match object.
Default to our logo if it doesn't exist tho ofc so this works while we're implementing the tournamentUrl thing.

Reply:
Does we also want to add custom background too ? As I see having ability to change the tournament image then it should be also possible to change the background image.

Reply:
In [a782ff2](https://github.com/ValoSpectra/Spectra-Frontend/pull/11/commits/a782ff291e8cd30d9eb637b24a74ab4f3c53d8de), I made the following logic:
* Set ```tournamentUrl``` to ```match.tournamentUrl``` if it exists and is not an empty string ```""```, otherwise default to ```"../../assets/misc/logo.webp"```. The same thing happens with ```tournamentBackgroundUrl```
* Change how the image preload in component.ts instead of using ```<link>``` HTML tag